src/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -75,6 +77,40 @@ public class SslStreamNetworkStreamTest
             listener.Stop();
         }
 
+        [Fact]
+        [OuterLoop] // Test hits external azure server.
+        public async Task SslStream_NetworkStream_Renegotiation_Succeeds()
+        {
+            Socket s = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+            await s.ConnectAsync(Configuration.Security.TlsRenegotiationServer, 443);
+            using (NetworkStream ns = new NetworkStream(s))
+            using (SslStream ssl = new SslStream(ns, true))
```

Comment:
Is there a reason why you are passing 'true' for the `SslStream` constructor `leaveInnerStreamOpen` parameter?  You could just use the default constructor which sets that variable to `false`.  Then, you wouldn't need to have a `using` clause for the `NetworkStream` 'ns' variable.

Reply:
Just to have a different kind of test code pattern. There are a lot of tests that pass false and exercise that code path.